src/app/public/modules/search/search.component.ts

```diff
@@ -70,31 +70,63 @@ const EXPAND_MODE_NONE: string = 'none';
 })
 export class SkySearchComponent implements OnDestroy, OnInit, OnChanges {
 
+/**
+ * Fires when the search text is applied.
+ */
   @Output()
   public searchApply = new EventEmitter<string>();
 
+/**
+ * Fires when the search text is changed.
+ */
   @Output()
   public searchChange = new EventEmitter<string>();
 
+/**
+ * Fires when the search text is cleared.
+ */
   @Output()
   public searchClear = new EventEmitter<void>();
 
+/**
+ * Specifies default search criteria for the input.
+ */
   @Input()
   public searchText: string;
 
+/**
+ * Specifies the expand mode for the search input. The valid options
+ * include `responsive` to collapse the search input into a button on
+ * mobile devices, `none` to *not* collapse the search input on mobile
+ * devices, and fit to extend the search input to `fit` the width of its container.
```

Comment:
Looks like we need to put the tick marks around the prior `fit` (instead of the latter).